channeldb/kvdb/config.go

```diff
@@ -17,6 +17,8 @@ type BoltConfig struct {
 
 // EtcdConfig holds etcd configuration.
 type EtcdConfig struct {
+	Embedded bool `long:"embedded" description:"Use embedded etcd instance instead of the external one."`
```

Comment:
It this only intended for test use? If so I would make that more clear. Maybe name it "TestInstance" or something.

Reply:
Can also be used to run an embedded instance than have _that_ connect out to an actual cluster of etcd nodes I believe?